models/user/setting_keys.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package user
+
+const (
+	// SettingsKeyHiddenCommentTypes is the settings key for hidden comment types
+	SettingsKeyHiddenCommentTypes = "issue.hidden_comment_types"
```

Comment:
I don't think we should use constants, at least, I haven't seen them being used in this way(translation key).

Reply:
It is not a translation key and it has been discussed before:

<img width="788" alt="image" src="https://user-images.githubusercontent.com/2114189/148624223-511584bc-0ae3-4ea0-b469-50ab51b585f1.png">


<img width="806" alt="image" src="https://user-images.githubusercontent.com/2114189/148624209-06548bf4-8c71-4571-85ba-ebdcf07ea154.png">